nab/detectors/context_ose/context_operator.py

```diff
@@ -0,0 +1,264 @@
+# -*- coding: utf-8 -*-

```

Comment:
FYI when this is merged I will add [the Numenta license comments](https://github.com/numenta/NAB/blob/master/nab/detectors/numenta/numenta_detector.py#L1-L19) above.